Explorer/Assets/DCL/Character/CharacterMotion/Platforms/PlatformSaveLocalRotation.cs

```diff
@@ -10,8 +10,26 @@ public static class PlatformSaveLocalRotation
         public static void Execute(ref CharacterPlatformComponent platformComponent, Vector3 forward)
         {
             if (platformComponent.CurrentPlatform != null)
-                platformComponent.LastAvatarRelativeRotation =
-                    platformComponent.CurrentPlatform.transform.InverseTransformDirection(forward);
+            {
+                Transform transform = platformComponent.CurrentPlatform.transform;
+                var currentPlatformRotation = transform.rotation;
+
+                if (platformComponent.LastPlatformRotation != null)
+                {
+                    float angleDifference = Quaternion.Angle(platformComponent.LastPlatformRotation.Value, currentPlatformRotation);
+
+                    if (angleDifference > Mathf.Epsilon)
+                        platformComponent.IsRotatingPlatform = true;
```

Comment:
```suggestion
                    platformComponent.IsRotatingPlatform = angleDifference > Mathf.Epsilon;
```

Maybe it's better to put it this way so that if a platform was rotating and stops rotating, the `IsRotatingPlatform` is put to `false` again, WDYT?